nmostesting/suites/BCP0060102Test.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (C) 2022 Advanced Media Workflow Association
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ..ControllerTest import ControllerTest, TestingFacadeException
+from ..NMOSUtils import NMOSUtils
+
+
+class BCP0060102Test(ControllerTest):
+    """
+    Runs Controller Tests covering BCP-006-01
+    """
+    def __init__(self, apis, registries, node, dns_server):
+        ControllerTest.__init__(self, apis, registries, node, dns_server)
+
+    def _generate_caps(self, media_types):
+        caps = {
+            "media_types": media_types, "constraint_sets": [
+                {
+                    "urn:x-nmos:cap:format:color_sampling": {
+                        "enum": ["YCbCr-4:2:2"]
+                    },
+                    "urn:x-nmos:cap:format:frame_height": {
+                        "enum": [1080]
+                    },
+                    "urn:x-nmos:cap:format:frame_width": {
+                        "enum": [1920]
+                    },
+                    "urn:x-nmos:cap:format:grain_rate": {
+                        "enum": [{
+                                "denominator": 1,
+                                "numerator": 25
+                        }]
+                    },
+                    "urn:x-nmos:cap:format:interlace_mode": {
+                        "enum": [
+                            "interlaced_bff",
+                            "interlaced_tff",
+                            "interlaced_psf"
+                        ]
+                    }
```

Comment:
Would it be as well to base these on `SDP_PARAMETERS`, so that at an event we can test Nodes and Controllers consistently with some event-wide operating point?